### ISSUE TITLE
CSS Alignment for Search Results

### ISSUE THREAD

**Issue by ronnieduke** [mautic/mautic#4758]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | 
| Feature request? | 
| Enhancement? | Y

## Description:
Seems like the alignment for the Name in the search results is off:

![image](https://user-images.githubusercontent.com/742579/29942863-2a12d868-8e4c-11e7-8a67-35ea59cbafe9.png)

adding a 10px top padding seems to center it:

```
#globalSearchDropdown .list-group-item>a{
    padding-top: 10px;
}
```
![image](https://user-images.githubusercontent.com/742579/29943051-caab3b08-8e4c-11e7-9db2-d9190d294488.png)

**Comment by escopecz** [comment]
Would you mind send this code change as a pull request?

**Comment by ronnieduke** [comment]
@escopecz - I'm a pro customer, so don't have the app and git setup locally...I'll see what I can do though.

**Comment by npracht** [comment]
Merged and fixed in 3.3.2 planed for March, 22nd!
